lib/internal/http2/core.js

```diff
@@ -1255,6 +1256,12 @@ function connectionListener(socket) {
     socket.on('timeout', socketOnTimeout);
   }
 
+  // TLS ALPN fallback to HTTP/1.1
+  if ((options.allowHTTP1 === undefined || !!options.allowHTTP1 === true) &&
```

Comment:
I think we should not coerce it for every connection. If we need to sanitize this value, we should do it elsewhere.